### PR TITLE
[FR] Update fr.yml

### PR DESCRIPTION
**Pull request by Wapdoowap**

### CODE REVIEW

File: fr.yml [hunk 1]
```diff
@@ -3,24 +3,24 @@ fr:
...
     error: "<strong>Kinky Opérateur :</strong> Doh… le code de vérification ci-dessus semble incorrect. Vérifiez s’il vous plaît que le numéro ci-dessus est identique à celui que vous avez reçu de nous, et ré-essayez."
     label: Code d’identification
     page_title: Validation à 2 facteurs
-    subtitle: 2-factorise-moi bébé !
+    subtitle: 2-factorise-moi bébé!
     title: Code secret
   about:
     contact_info: Informations de contact
-    email_us: Envoyez-nous un courriel à <a href="mailto:support@fetlife.com">support@fetlife.com</a> et nous reviendrons très vite vers vous !
+    email_us: Envoyez-nous un e-mail à <a href="mailto:support@fetlife.com">support@fetlife.com</a> et nous reviendrons très vite vers vous !
```
Comment (lucsurr) [inline fr.yml]: We typically say 'adresse mail', not 'adresse e-mail' (and we send a 'mail' too…).

File: fr.yml [hunk 2]
```diff
@@ -62,33 +62,33 @@ fr:
     campaigns: Campagnes
     clicks: Clics
     ctr: CTR
-    edit: modifier
+    edit: Modifier
```
Comment (lucsurr) [inline fr.yml]: Would stick to following ref file (i.e. `en.yml`) for capitalization of first letter (i.e. if first letter is not capitalized in English, then do not do it in French, and if you think this is an error, then first submit fix to English version).

File: fr.yml [hunk 3]
```diff
@@ -62,33 +62,33 @@ fr:
...
+    view: Vue
     worst: Pire
   feed:
     comment: Commentaire
     comments_count:
       one: 1 commentaire
       other: "%{count_with_delimiter} commentaires"
     love: Aimer
-    loved_hover: Bourreau des cœurs ! :-(
+    loved_hover: Bourreau des cœurs! :-(
```
Comment (lucsurr) [inline fr.yml]: This is incorrect, please keep unbreakable spaces before high punctuation marks (? ! ; : ).

File: fr.yml [hunk 4]
```diff
@@ -62,33 +62,33 @@ fr:
...
     loves_count:
       one: 1 amour
       other: "%{count_with_delimiter} amours"
     modal:
-      close: fermer
-      more: plus
+      close: Fermer
+      more: Pus
```
Comment (lucsurr) [inline fr.yml]: Typo (I know it’s not easy to proof-read yourself) (`plus` → `Pus`…)

File: fr.yml [hunk 5]
```diff
@@ -99,50 +99,50 @@ fr:
...
+      error_country_code: "<strong>Opérateur Kinky:</strong> Assurez-vous de commencer votre numéro par votre code de pays : +1 pour les États Unis et le Canada, +33 pour la France, +32 pour la Belgique, +44 pour le Royaume Uni, +61 pour l’Australie, +31 pour les Pays-Bas, +49 pour l’Allemagne, etc."
+      error_quota: "<strong>Opérateur Kinky:</strong> Aïe! Nous ne pouvons essayer d’envoyer que 3 textos par 24 heures à un même numéro mobile. Que le numéro existe dans notre base de données ou non."
+      pro_tip: "<strong>Astuce de pro:</strong> Assurez-vous de commencer votre numéro par votre code de pays : +1 pour les États Unis et le Canada, +33 pour la France, +32 pour la Belgique, +44 pour le Royaume Uni, +61 pour l’Australie, +31 pour les Pays-Bas, +49 pour l’Allemagne, etc."
       step1:
-        page_title: Réinitialiser le mot de passe via texto
-        subtitle: Laissez-moi vous aider avec ça.
-        title: Oublié votre mot de passe ?
+        page_title: Réinitialiser le mot de passe via texto.
```
Comment (lucsurr) [inline fr.yml]: Same as with initial capital, let's not add final dots if they are not present in English...

**Comment by lucsurr** [comment]
Embedded some notes as comments.

Also, you'll need to update your branch, they changed the layout of the repo (translation files are now in `locales/` sub-directory).

As a side note, would recommend to make smaller, more focused patches (tackling either a single cleanup topic, like one for tu → vous, one for email, etc., or only updating a few sections of the translation). Would be simpler to manage imho, git is collaborative tool, it does not work that well with massive changes in single commit…

File: fr.yml (hunk 2 above)
Comment (reneklacan) [inline fr.yml]: +1

File: fr.yml (hunk 1 above)
Comment (yianni3000) [inline fr.yml]: Nice improvement here, did not know that @lucsurr !

File: fr.yml (hunk 2 above)
Comment (yianni3000) [inline fr.yml]: I also agree with the capitalization strategy here @lucsurr , nice

File: fr.yml (hunk 3 above)
Comment (yianni3000) [inline fr.yml]: Ok!! Unbreakable spaces it is. Yet another thing I did not know :/

File: fr.yml (hunk 4 above)
Comment (yianni3000) [inline fr.yml]: I was sure it was plus! Nice catch.

File: fr.yml (hunk 5 above)
Comment (yianni3000) [inline fr.yml]: Agreed once again with final dots strategy @lucsurr , great great stuff.

**Comment by yianni3000** [comment]
Hey @joaocunha ! Can you please check this out?

**Comment by joaocunha** [comment]
@Wapdoowap you have to fix the conflicts on your branch before trying to merge. Do you know how to do so?

**Comment by yianni3000** [comment]
@Wapdoowap  Great great contributions and help. Thank you again, boom!

File: fr.yml (hunk 3 above)
Comment (Wapdoowap) [inline fr.yml]: Oh, sorry, I thought they were breakable spaces, which would have been a mess... I agree with the unbreakable spaces, they're the recommended form.

File: fr.yml (hunk 1 above)
Comment (Wapdoowap) [inline fr.yml]: I agree re: common use (in France / Belgium), but I was trying to harmonize the entire file to not have three different forms (mail, e-mail, courriel)... I do think it's better to say "adresse mail", indeed.

**Comment by reneklacan** [comment]
Looks like we forgot about this and now differences are too big :-/ Sorry

**Pull request closed without merging**